test/integration/targets/tower_settings/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+- name: Set the value of AWX_PROOT_SHOW_PATHS
+  tower_settings:
+    name: "AWX_PROOT_SHOW_PATHS"
+    value: "'/var/lib/awx/projects/', '/tmp'"
+  register: result
+
+- assert:
+    that:
+      - "result is changed"
+
+- name: Set the value of AWX_PROOT_BASE_PATH
+  tower_settings:
+    name: AWX_PROOT_BASE_PATH
+    value: "/tmp"
+  register: result
+
+- name: Get the value of AWX_PROOT_BASE_PATH
+  tower_settings:
+    name: AWX_PROOT_BASE_PATH
+  register: result
+
+- assert:
+    that:
+      - "result.value == '/tmp'"
```

Comment:
These tests actually bring up good question - is there any way to reset the setting back to the default? I don't think it's likely to matter for this particular settings & value, but you could hypothetically interfere with subsequent tests by leaving the value set to this.

Reply:
I can do that. Before running these tests I can get the value using the same module. Once these tests are done set it back to what it was before. Is that good ?

Reply:
> Before running these tests I can get the value using the same module. Once these tests are done set it back to what it was before. Is that good ?

Again, it's not super important as a practical matter for running the tests, however, I think this would be a good example to set.

Even though you can reset a setting back to its default via the API, it doesn't really make the most sense for the modules, because as you start doing something, it could already have an existing non-default value. Saving the prior value and then re-setting it to that value is the best behavior.

Reply:
@AlanCoding Thanks for the review. I have made this change in tests.